src/main/kotlin/gdx/liftoff/data/platforms/Lwjgl3.kt

```diff
@@ -214,7 +214,7 @@ construo {
     // human-readable name, used for example in the `.app` name for macOS
     humanName.set(appName)
     // Optional, defaults to project version
-    version.set("0.0.0")
```

Comment:
Good catch; a fair bit of Construo configuration didn't get updated during the process of bringing it over...